Psychtoolbox/PsychHardware/NetStation.m

```diff
@@ -79,13 +100,17 @@
 %   This function was modified by Matt Mollison to accommodate sending more than just int16s to Net Station
 %   Improved by Justin Ales 2014.
 %   Consolidated by Gergely Csibra, 2015
+%   Improved by Justin Ales 2017.
+%   getntpsynchronization added by Andreas Widmann, 2017
 %%
 
 persistent NSIDENTIFIER;
 persistent NSSTATUS;
 persistent NSRECORDING;
 persistent SYNCHEPOCH; %This is to fix the bug that large epochs (like unix) overflows the int32 value --JMA
 
+persistent NTPSOCK GETNTPSYNCED NTPLOG x;
```

Comment:
Not clear to me if 'x' needs to be persistent? There are two x = linreg() calls, but only the result of one seems to be used, the other seems to have no meaning, as x always gets recalculated before use? If this is correct then x shouldn't be x, but some descriptive name, also in ALLCAPS like all other persistent variables/pseudo-constants.